processing file asmFiles/test.associativity.asm (1 of 1)
processing file asmFiles/test.dhit_ihit_same_cycle.asm (1 of 1)
processing file asmFiles/test.ihit_before_dhit.asm (1 of 1)
--- memsim.hex	2024-10-18 11:32:33.510904905 -0400
+++ memcpu.hex	2024-10-18 11:32:33.495904322 -0400
@@ -11,4 +11,5 @@
 :04003C000000733716
 :04003D000000270197
 :04003E000000123478
+:040C4000FFFFFFFFB4
 :00000001FF
processing file asmFiles/test.associativity.asm (1 of 1)
processing file asmFiles/test.ihit_before_dhit.asm (1 of 1)
processing file asmFiles/test.dhit_ihit_same_cycle.asm (1 of 1)
processing file asmFiles/count_days.asm (1 of 34)
processing file asmFiles/fib.asm (2 of 34)
processing file asmFiles/mergesort.asm (3 of 34)
--- memsim.hex	2024-10-18 11:57:03.783025432 -0400
+++ memcpu.hex	2024-10-18 11:57:03.760024537 -0400
@@ -215,6 +215,6 @@
 :04017D00000000631B
 :04017E000000006419
 :04017F000000006418
-:040C40000000057437
+:040C40000000057239
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mult.asm (4 of 34)
processing file asmFiles/mult_procedure.asm (5 of 34)
processing file asmFiles/search.asm (6 of 34)
processing file asmFiles/test.associativity.asm (7 of 34)
processing file asmFiles/test.beq_ntaken.asm (8 of 34)
processing file asmFiles/test.beq_taken.asm (9 of 34)
processing file asmFiles/test.bge_ntaken.asm (10 of 34)
processing file asmFiles/test.bge_taken.asm (11 of 34)
processing file asmFiles/test.bgeu_ntaken.asm (12 of 34)
processing file asmFiles/test.bgeu_taken.asm (13 of 34)
processing file asmFiles/test.blt_ntaken.asm (14 of 34)
processing file asmFiles/test.blt_taken.asm (15 of 34)
processing file asmFiles/test.bltu_ntaken.asm (16 of 34)
processing file asmFiles/test.bltu_taken.asm (17 of 34)
processing file asmFiles/test.bne_ntaken.asm (18 of 34)
processing file asmFiles/test.bne_taken.asm (19 of 34)
processing file asmFiles/test.branchjump1.asm (20 of 34)
processing file asmFiles/test.cfhazard_beq.asm (21 of 34)
processing file asmFiles/test.dhazard_add.asm (22 of 34)
processing file asmFiles/test.dhazard_lw.asm (23 of 34)
processing file asmFiles/test.dhit_ihit_same_cycle.asm (24 of 34)
processing file asmFiles/test.halt.asm (25 of 34)
processing file asmFiles/test.ihit_before_dhit.asm (26 of 34)
processing file asmFiles/test.jal.asm (27 of 34)
processing file asmFiles/test.jalr.asm (28 of 34)
processing file asmFiles/test.ldst1.asm (29 of 34)
processing file asmFiles/test.ldst2.asm (30 of 34)
processing file asmFiles/test.loadstore.asm (31 of 34)
processing file asmFiles/test.mergesort_debug.asm (32 of 34)
processing file asmFiles/test.rtype.asm (33 of 34)
processing file asmFiles/test.shazard_lw.asm (34 of 34)
